### ISSUE TITLE
Cosmetic: use PyTorch's DataLoader

### ISSUE THREAD

**Issue by mkroutikov** [jiesutd/NCRFpp#89]
Use DataLoader to do shuffling and batching:

https://pytorch.org/docs/stable/data.html

**Comment by jiesutd** [comment]
Thanks for the suggestion! Yes, I have the plan to use the DataLoader for the data preparing. But current I do not have enough time working on this.
